docs/isoSPI.md

```diff
@@ -1,3 +1,109 @@
-# AMDC isoSPI Interface
+# AMDC isoSPI and Differential I/O Interface

```

Comment:
This file has evolved to documenting the entire expansion port (isoSPI + differential I/I interface). I think that is a good change. Please reflect that change in the file name (change `isoSPI.md` to be `expansionPort.md`)

Reply:
Renamed the file as per the suggestion